products/products.controller.ts

```diff
@@ -0,0 +1,33 @@
+import { Controller,Post,Body,Get,Param,Patch ,Delete} from "@nestjs/common";
+import { productService } from "./products.service";
+@Controller('products')
+export class ProductsController{
+  constructor(private readonly productService:productService){}
+    @Post()
+    addProdcut(@Body('title') prodTitle:string,@Body('description') proddesc:string,@Body('price') prodprice:number){
```

Comment:
use DTO to get the field from the request body. and format the code using prettier.